aqandu/__init__.py

```diff
@@ -1,26 +1,35 @@
-from assets import init
+import assets
 import config
-from os import getenv, environ
+import os
 from dotenv import load_dotenv
 from flask import Flask
-from flask_caching import Cache
-from google.cloud.bigquery import Client
+from google.cloud import bigquery
+import logging
+import time
+
 
 load_dotenv()
-PROJECT_ID = getenv("PROJECTID")
+PROJECT_ID = os.getenv("PROJECTID")
+
+
+logfile = "serve.log"
+logging.basicConfig(filename=logfile, level=logging.DEBUG, format = '%(levelname)s: %(filename)s: %(message)s')
+logging.info('API server started at %s', time.asctime(time.localtime()))
+logging.debug('This message should go to the log file')
+logging.warning('And this, too')
+logging.error('And non-ASCII stuff, too, like Øresund and Malmö')
+
 
 app = Flask(__name__)
 app.config.from_object(config)
-app.config["CACHE_TYPE"] = "simple"
-app.config["CACHE_DEFAULT_TIMEOUT"] = 1
-init(app)
-
-cache = Cache(app)
+assets.init(app)
```

Comment:
Please keep the caching
```suggestion
app.config["CACHE_TYPE"] = "simple"
app.config["CACHE_DEFAULT_TIMEOUT"] = 1
cache = Cache(app)

assets.init(app)
```